acta-ornithologica.csl

```diff
@@ -0,0 +1,143 @@
+  <macro name="volume">
+    <group delimiter=" ">
+      <label text-case="capitalize-first" variable="volume" form="short"/>
+      <number variable="volume"/>
+    </group>
+  </macro>
+  <macro name="pages">
+    <group delimiter=" ">
+      <label variable="page" form="short"/>
+      <text variable="page"/>
+    </group>
+  </macro>
+  <citation disambiguate-add-year-suffix="true" collapse="year" et-al-min="3" et-al-use-first="1">
+    <sort>
+      <key macro="year"/>
+      <key variable="author"/>
+    </sort>
+    <layout delimiter="; " prefix="(" suffix=")">
+      <group delimiter=" ">
+        <names variable="author" suffix=", ">
+          <name and="symbol" delimiter=" " name-as-sort-order="all" sort-separator=", " form="short"/>
+        </names>
+        <date variable="issued">
+          <date-part name="year" form="long"/>
+        </date>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true">
+    <sort>
+      <key macro="author"/>
+      <key macro="year" sort="ascending"/>
+    </sort>
+    <layout suffix=".">
+      <group delimiter=". ">
+        <text macro="author"/>
+        <text macro="year"/>
+        <choose>
+          <if type="thesis">
+            <text macro="title" font-style="italic"/>
+            <text variable="publisher" suffix=", "/>
+            <text variable="publisher-place"/>
+          </if>
+          <else-if type="book">
+            <group delimiter=". ">
+              <text macro="title" font-style="italic"/>
+              <text macro="volume"/>
+              <text variable="publisher"/>
+            </group>
+          </else-if>
+          <else-if type="chapter">
+            <group delimiter=". ">
+              <text macro="title"/>
+              <group delimiter=": ">
+                <text term="in" text-case="capitalize-first"/>
+                <text macro="editor"/>
+              </group>
+              <text variable="container-title"/>
+              <group delimiter=", ">
+                <text variable="publisher"/>
+                <text macro="pages"/>
+              </group>
+            </group>
+          </else-if>
+          <else-if type="paper-conference">
+            <text macro="title"/>
+            <text variable="event" prefix=" " suffix=", " font-style="italic"/>
+            <text variable="volume" suffix=", "/>
+            <text variable="page" suffix="."/>
+          </else-if>
+          <else-if type="webpage post post-weblog" match="any">
+            <group delimiter=". ">
+              <text variable="container-title"/>
+              <group delimiter=": ">
+                <text term="available at" text-case="capitalize-first"/>
+                <text variable="URL"/>
+              </group>
+            </group>
+          </else-if>
+          <else>
+            <group delimiter=". ">
+              <text macro="title"/>
+              <text variable="container-title" form="short" strip-periods="true"/>
```

Comment:
I don't think this is right: The journal abbreviations are _with_ period but there is _no_ period after the journal name, see e.g. `Zool. Zhurnal 72: 85–97.` from the example article

Reply:
Yeah, thanks for spotting that.
I looked at "Acta Ornithol." and didn't get that Acta is a full word...